daemons/controld/controld_fsa.c

```diff
@@ -184,7 +183,8 @@ s_crmd_fsa(enum crmd_fsa_cause cause)
         fsa_dump_actions(fsa_data->actions, "Restored actions");
 
         /* get the next batch of actions */
-        new_actions = crmd_fsa_actions[fsa_data->fsa_input][globals->fsa_state];
+        new_actions = controld_fsa_get_action(fsa_data->fsa_input,
+                                              globals->fsa_state);
```

Comment:
Also not for this PR, but the second argument is unnecessary

Reply:
I mean the function already has access to globals->fsa_state

Reply:
D'oh. Yeah if this is remains the only caller, then we don't need argument 2 for either accessor. (And if it doesn't, we can re-add the arg.)